README.md

```diff
@@ -1,3 +1,11 @@
+
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Run tests](https://img.shields.io/badge/Run%20tests-passing-green)
```

Comment:
```suggestion
```
Until we have Travis CI running tests, its best to take this out. We are waiting for this issue https://github.com/anitab-org/mentorship-android/issues/704 to be fixed so that we can run tests on the build.